app/views/cloud_volume/attach.html.haml

```diff
@@ -57,4 +57,5 @@
 :javascript
   ManageIQ.angular.app.value('cloudVolumeFormId', '#{@volume.id}');
   ManageIQ.angular.app.value('storageManagerId', #{@volume.ext_management_system.id});
```

Comment:
@himdel in Fine the IDs seem to be integers so we don't need to change to string as on the original PR.

Reply:
We definitely do, the same issue with too big integers is also present in fine

(Furthermore, you may need to do the `from_cid` / `to_cid` trasnformation - you need to use a region other than 0 in your database in order to make sure this works.)

Reply:
(IIRC in regions other than 0, the ids will come in the `10r256` format from the API too)